### PR TITLE
Pass a11y states from greeter to session

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: data/50-org.ayatana.indicator.a11y.AccountsService.pkla [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Allow LightDM to set AccountsService fields]
```
Comment (sunweaver) [inline data/50-org.ayatana.indicator.a11y.AccountsService.pkla:1]: @tari01 you are aware that .pkla files are deprecated in policykit? We need to move forward here across all indicators, probably.

Comment (sunweaver) [reply data/50-org.ayatana.indicator.a11y.AccountsService.pkla:1]: But not in this PR.

File: src/service.c [hunk 2]
```diff
@@ -197,6 +291,18 @@ static void onDispose (GObject *pObject)
             g_free (self->pPrivate->sThemeIcon);
         }
     }
+    else
+    {
+        if (self->pPrivate->nArcticaSubscription)
```
Comment (sunweaver) [inline src/service.c:296]: I'd love to see the "Arctica" name not be so prominent in the code. GreeterSubscription, maybe?

In fact I have been thinking of renaming Arctica Greeter to Ayatana Greeter several times in the past, so we should not be to Arctica-centric here.

**Review by sunweaver — COMMENTED** [review]
See my inline comments.

File: data/50-org.ayatana.indicator.a11y.AccountsService.pkla (hunk 1 above)
Comment (tari01) [reply data/50-org.ayatana.indicator.a11y.AccountsService.pkla:1]: Yes, there is an issue opened for this already. I keep skipping it, but will eventually get there.

File: src/service.c (hunk 2 above)
Comment (tari01) [reply src/service.c]: Done.

File: src/service.c [hunk 3]
```diff
@@ -16,10 +16,14 @@
...
 #include <glib/gi18n.h>
 #include <gio/gio.h>
+#include <act/act.h>
 #include "service.h"
 
 #define BUS_NAME "org.ayatana.indicator.a11y"
 #define BUS_PATH "/org/ayatana/indicator/a11y"
+#define GREETER_BUS_NAME "org.ArcticaProject.ArcticaGreeter"
+#define GREETER_BUS_PATH "/org/ArcticaProject/ArcticaGreeter"
+#define GREETER_SETTINGS "org.ArcticaProject.arctica-greeter"
```
[the remark this answers is not in the document]
Comment (tari01) [reply src/service.c:26]: Done.

**Review by sunweaver — APPROVED** [review]
ACK. Looking forward to test this in more depth.

**Pull request merged**